storage/api/Storage.Samples/MakePublic.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2020 Google Inc.
+﻿// Copyright 2021 Google Inc.
```

Comment:
Please revert this one, as this sample was written in 2020. We keep the copyright year of when we added the file, not when we last modified it

Reply:
Done